firebase-ai-friendly-meals/apple/FriendlyMeals/FriendlyMeals/Features/MealPlannerChat/MealPlannerChatViewModel.swift

```diff
@@ -238,4 +193,25 @@ class MealPlannerChatViewModel {
       messages.append(errorMessage)
     }
   }
+  
+  private func handleFunctionCall(_ call: FunctionCallPart) -> FunctionResponsePart? {
+    if call.name == "startTimer" {
```

Comment:
very slight nit: you can use `switch (call.name)` here for a marginally more aesthetic implementation.